inc/staticmisc.class.php

```diff
@@ -157,7 +157,7 @@ static function getModulesMethods() {
    static function task_definitiontype_wakeonlan($a_itemtype) {
 
       $a_itemtype['Computer'] = Computer::getTypeName();
-      $a_itemtype['PluginFusioninventoryDeployGroup'] = __('Dynamic Group');
+      $a_itemtype['PluginFusioninventoryDeployGroup'] = __('Fusioninventory group');
```

Comment:
'Fusioninventory group' is not in GLPI locales context, so you should add ``, 'fusioninventory'``

Reply:
hi @ddurieux ! can you perform the requested change ? thanks

Reply:
I suggest to modify the locale in DeployGroup::getTypeName and use this method instead of calling directly gettext